charts/datadog/templates/cluster-agent-rbac.yaml

```diff
@@ -66,6 +66,14 @@ rules:
   verbs:
   - get
   - update
+- apiGroups:
+  - "coordination.k8s.io"
+  resources:
+  - leases
+  resourceNames:
+  - {{ template "datadog.fullname" . }}-leader-election  # Leader election token
+  verbs:
+  - get
```

Comment:
You probably also need `update` and `create`.

Reply:
It should be done